README.md

```diff
@@ -1,26 +1,33 @@
-# Export Insurance Policies
+# Export Insurance Policies (EXIP) :briefcase:
+Welcome to the Export Insurance Policies repository! This repository houses the code for the UK Export Finance (UKEF) EXIP service, a public-facing application that offers various features to exporters:
 
-This repository contains the code for the UK Export Finance EXIP service - a public facing application that allows exporters to:
+- Obtain a quote for export insurance coverage with UKEF.
```

Comment:
Should we be aligned with the actual service name here (before this change it's "Get a quote", same as the service name)